src/index.html

```diff
@@ -1,18 +1,70 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <section class="product-cards">
+      <article
+        class="product-card"
+        data-qa="card"
+      >
+        <p class="product-card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="product-card__code-product">Product code: 195434</p>
+
+        <div class="product-card__reviews">
+          <div class="product-card__reviews-stars product-card__reviews-stars--4">
```

Comment:
BEM modifier for 'product-card__reviews-stars' should be used differently. Instead of creating a new class for each number of active stars, consider using a modifier like 'product-card__reviews-stars--4' to indicate the number of active stars and style them accordingly using CSS.